public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -96,6 +97,19 @@ export function panelMenuBehavior(menu: VizPanelMenu, isRepeat = false) {
           dashboard.showModal(drawer);
         },
       });
+      subMenu.push({
+        text: t('share-panel.menu.share-embed-title', 'Share embed'),
```

Comment:
Is `Share embed` clear enough? In the other cases (Share Link, Share Snapshot) the Sharing action or resource is clear.
Maybe we could use "Embed Panel" in this case or something like that.

Reply:
I think it makes sense when talking about the menu title, *but* I was wondering the same thing when it refers to the Drawer title, taking into account the Drawer is open with full screen height and maybe the user losses context and doesn't know what's been shared. Maybe "Share Embed Panel" would make sense. @natydej ?

Reply:
I think we should be consistent with the title option that we use on the dropdown, in this case "Share embed" should enough, people already know that they are coming from a panel flow.
Here are the [figma link](https://www.figma.com/design/AjvrAorYpnfSoMpU5kwf39/Adoption-of-Sharing?node-id=4560-57369&t=fl7JaNQzod3sIVbL-1) for designs
<img width="387" alt="Screenshot 2024-08-02 at 10 58 11" src="https://github.com/user-attachments/assets/735cc0b9-b07e-4d47-a631-f65005bb49fa">
<img width="634" alt="Screenshot 2024-08-02 at 10 58 00" src="https://github.com/user-attachments/assets/cb3c975b-b475-4458-8756-9d6699ea29c3">